### PR TITLE
Add google analytics

### PR DESCRIPTION
**Pull request by denebtech**

### CODE REVIEW

File: docker/env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# this is the postgres user's (admin) password
+POSTGRES_PASSWORD=mysecretpassword
```
Comment (brycemcd) [inline docker/env]: This file should be part of the `.gitignore`. We don't want this file in the repo

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -7,6 +7,21 @@
...
     <link rel="stylesheet" href="https://unpkg.com/react-select/dist/react-select.css">
+
+     <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src= <%= "https://www.googletagmanager.com/gtag/js?id=" + ENV["google_analytics_key"]  %>>
+
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+      gtag('config', <%= ENV["google_analytics_key"] %>);
```
Comment (brycemcd) [inline app/views/layouts/application.html.erb]: Does this get rendered as a string (i.e. "foo") or as a variable (i.e. `foo`)?

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -7,6 +7,21 @@
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application' %>
     <link rel="stylesheet" href="https://unpkg.com/react-select/dist/react-select.css">
+
+     <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src= <%= "https://www.googletagmanager.com/gtag/js?id=" + ENV["google_analytics_key"]  %>>
```
Comment (brycemcd) [inline app/views/layouts/application.html.erb]: Why not move the string part out into it's own part and use the ERB wrapper around the ruby part? i.e. `src="googletagmanager.com/gtag/js?id=<%= ENV['google_analytics_key']%>"`

File: docker/env (hunk 1 above)
Comment (andrezafeu) [reply docker/env]: @brycemcd I think we should also add logfile to .gitignore

Comment (denebtech) [reply docker/env]: I just added `!/log/.keep` to .gitignore.

Comment (brycemcd) [reply docker/env]: please remove this file from the PR

**Comment by brycemcd** [comment]
Please remove that file and then :shipit: (merge to master). LGTM

**Pull request merged**